docs/learn/guides/services/restful-service/exposing-a-soap-service/README.md

```diff
@@ -234,8 +234,10 @@ service stockQuote on httpListener {
 
 
 ## Testing 
-### Running the axis2 server
-First run the axis2 server by following the [documentation]().
+### Running the axis2 server and deploying the back-end service
```

Comment:
We may not need this README since we've re-written the guide with inbuilt mock soap service with https://github.com/wso2/ballerina-integrator/pull/399